package.json

```diff
@@ -19,7 +19,7 @@
   "author": "Relaying Services Team",
   "license": "MIT",
   "dependencies": {
-    "@rsksmart/rif-relay-common": "https://github.com/JONAF2103/rif-relay-common",
+    "@rsksmart/rif-relay-common": "https://github.com/infuy/rif-relay-common",
```

Comment:
i think `JONAF2103` will redirect but it's still better to have this changed :+1: